<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>962</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(242, 242, 242);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(242, 242, 242);</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="baseSize">
               <size>
                <width>240</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>30</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(91, 91, 91);</string>
              </property>
              <property name="text">
               <string>   환자 관리 SW</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>738</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="enrollFrame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>110</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>110</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>200</width>
                <height>110</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QPushButton" name="enrollButton">
               <property name="geometry">
                <rect>
                 <x>50</x>
                 <y>20</y>
                 <width>131</width>
                 <height>71</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, 
                                                        y2:1, stop:0 rgba(149, 210, 253, 255),  stop:1 rgba(152, 136, 248, 255));
border-radius:10px;
color:#ffffff;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/img/img/enroll.png</normaloff>:/img/img/enroll.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QFrame" name="patientInfoFrame">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 30px;
</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::WinPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QFrame" name="patientStatusFrame">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>2</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>230</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>230</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 30px;
</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="medicalRecordFrame">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="baseSize">
                   <size>
                    <width>110</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 30px;
</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::WinPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QFrame" name="imageFrame">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>500</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>340</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 30px;
</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::WinPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>270</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>570</width>
              <height>360</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>360</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>185</x>
               <y>191</y>
               <width>81</width>
               <height>35</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="text">
              <string>Port:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_12">
             <property name="geometry">
              <rect>
               <x>4</x>
               <y>4</y>
               <width>541</width>
               <height>331</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 40px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_3">
             <property name="geometry">
              <rect>
               <x>185</x>
               <y>144</y>
               <width>81</width>
               <height>35</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>20</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="text">
              <string>IP:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_13">
             <property name="geometry">
              <rect>
               <x>25</x>
               <y>24</y>
               <width>541</width>
               <height>331</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(196, 196, 196);
border-radius: 40px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLineEdit" name="portLineEdit_4">
             <property name="geometry">
              <rect>
               <x>275</x>
               <y>194</y>
               <width>211</width>
               <height>35</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="ipLineEdit_4">
             <property name="geometry">
              <rect>
               <x>275</x>
               <y>144</y>
               <width>211</width>
               <height>35</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="ipLineEdit">
             <property name="geometry">
              <rect>
               <x>285</x>
               <y>146</y>
               <width>191</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>17</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 170, 0, 00);
color: rgb(100, 100, 100);
border: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="text">
              <string>192.168.0.10</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_10">
             <property name="geometry">
              <rect>
               <x>51</x>
               <y>40</y>
               <width>231</width>
               <height>50</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>맑은 고딕</family>
               <pointsize>23</pointsize>
               <italic>false</italic>
               <bold>true</bold>
               <strikeout>false</strikeout>
               <kerning>true</kerning>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255, 255);
color: rgb(255, 162, 0);
border-radius: 15px;</string>
             </property>
             <property name="text">
              <string>환자관리 SW</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
            <widget class="QLabel" name="label_11">
             <property name="geometry">
              <rect>
               <x>65</x>
               <y>134</y>
               <width>100</width>
               <height>100</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/img/img/login.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="portLineEdit">
             <property name="geometry">
              <rect>
               <x>285</x>
               <y>196</y>
               <width>191</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>17</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 170, 0, 00);
color: rgb(100, 100, 100);
border: rgba(255, 255, 255, 0);</string>
             </property>
             <property name="text">
              <string>8000</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>410</x>
               <y>260</y>
               <width>81</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:10px;
color: rgb(255, 255, 255);

background-color: rgb(249, 117, 23);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton">
             <property name="geometry">
              <rect>
               <x>405</x>
               <y>254</y>
               <width>81</width>
               <height>51</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border-radius:10px;
background-color: rgb(255, 162, 0);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>접속</string>
             </property>
            </widget>
            <zorder>label_13</zorder>
            <zorder>label_12</zorder>
            <zorder>label_2</zorder>
            <zorder>label_3</zorder>
            <zorder>portLineEdit_4</zorder>
            <zorder>ipLineEdit_4</zorder>
            <zorder>ipLineEdit</zorder>
            <zorder>label_10</zorder>
            <zorder>label_11</zorder>
            <zorder>portLineEdit</zorder>
            <zorder>pushButton_2</zorder>
            <zorder>pushButton</zorder>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>270</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>화면 초기화</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>월간 일정</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>도움말</string>
   </property>
  </action>
  <action name="actionInitialize">
   <property name="text">
    <string>화면 초기화</string>
   </property>
  </action>
  <action name="actionMonthlyReservation">
   <property name="text">
    <string>월간 예약</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
